CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* DataCollector
+  * Fix memory leak from `PayPalDataCollector` (fixes #419)
```

Comment:
Any ideas for the best CHANGELOG entry here? The method where we call the magnes SDK is `PayPalDataCollector#getClientMetadataId()`, but that class is internal and is used in Local Payment, PayPal, and DataCollector. Should all those be called out here?

Reply:
I think what you have is fine!